packages/@aws-cdk/aws-codepipeline-actions/README.md

```diff
@@ -689,6 +689,80 @@ const deployStage = pipeline.addStage({
 });
 ```
 
+#### Invalidating Cloudfront Cache when deploying to S3
+
+There is currently no support in the S3DeployAction construct to invalidate the Cloudfront cache after upload.
+
+One workaround is to add another build step after the deploy step, to use the AWS CLI to invalidate the cache.
+
+```ts
+// Create a Cloudfront Web Distribution
+const cfDist = new cloudfront.CloudFrontWebDistribution(this, `CloudFrontDistribution`, {
+  originConfigs: [
+    {
+      // ...
+      behaviors : [ {isDefaultBehavior: true}]
+    },
+    // ...
+  ],
+});
```

Comment:
Let's just leave all of this empty - this doc is about CodePipeline, not really about CloudFront 🙂.

```suggestion
 // ...
});
```